client/src/reducers/auth.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  REGISTER_SUCCESS,
+  REGISTER_FAILURE,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+  LOG_OUT,
+  AUTH_ERROR,
+  USER_LOADED,
+} from "../actions/types";
+
+interface InitialState {
+  token: string | any;
+  isAuthenticated: boolean;
+  loading: boolean;
+  user: null | any;
```

Comment:
Try to avoid `any` and create interface for user

Reply:
done